TipCatDotNet.Api/Controllers/AccountController.cs

```diff
@@ -77,6 +82,25 @@ public async Task<IActionResult> Update([FromRoute] int accountId, [FromBody] Ac
     }
 
 
+    /// <summary>
+    /// Gets default amount of transactions per facilities by current account.
+    /// </summary>
+    /// <param name="filterProperty">The transaction's property by which it filters</param>
+    /// <returns></returns>
+    [HttpGet("/current/transactions")]
+    [ProducesResponseType(typeof(List<FacilityTransactionResponse>), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    public async Task<IActionResult> GetDefaultPerFacilities([FromQuery] TransactionFilterProperty filterProperty = TransactionFilterProperty.CreatedDESC)
```

Comment:
This one should be similar to account members. Could names `GetTransactions` and contains paging as well